### DISCUSSION TITLE
Manually decrypt the bookmarks.xbel file from Google Drive?

### DISCUSSION THREAD

**Discussion by akash07k** [floccusaddon/floccus#1711] [Q&A]
Hi all,
So, I have my `bookmarks.xbel` file on Google Drive, which I have downloaded to my local file system. I also have the correct password for it. However, for some unknown reason, Floccus is not recognizing any bookmarks in it.
It seems that the file indeed contains many bookmarks, as the file size is 558 kilobytes. Now, I want to manually decrypt this file with the password to see what's inside. I have many bookmarks that I can't afford to lose, so manual decryption would be very helpful if I can find the bookmarks in it by any chance.
Can someone please help me with the steps for the decryption? I'm not even sure which encryption method and system Floccus supports. Please help me.

**Comment by marcelklehr** [comment]
Hi @akash07k 
To my knowledge there is no tool that can directly decrypt the files floccus produces. Floccus uses standard cryptographic primitives but the format is not standardized. Your best bet, I think, is to try and make floccus decrypt it. could you let me know which error message you are seeing?

**Reply by akash07k** [reply]
Actually there is no error message.
Just Floccus is not seing any bookmarks in the file.

> Hi @akash07k To my knowledge there is no tool that can directly decrypt the files floccus produces. Floccus uses standard cryptographic primitives but the format is not standardized. Your best bet, I think, is to try and make floccus decrypt it. could you let me know which error message you are seeing?
